projects/storefrontlib/cms-components/navigation/navigation/navigation-ui.component.html

```diff
@@ -102,26 +102,50 @@
           </button>
         </ng-container>
         <ng-container *cxFeature="'a11yNavigationUiKeyboardControls'">
-          <button
-            aria-level="4"
-            [attr.role]="(isDesktop$ | async) && depth ? 'heading' : 'button'"
-            [attr.aria-haspopup]="true"
-            [attr.aria-expanded]="false"
-            [attr.aria-controls]="node.title"
-            [attr.aria-describedby]="'greeting'"
-            (click)="toggleOpen($any($event))"
-            (mouseenter)="onMouseEnter($event)"
-            (keydown.space)="onSpace($any($event))"
-            (keydown.enter)="onSpace($any($event))"
-            (keydown.esc)="back()"
-            (keydown.arrowDown)="focusOnNode($any($event))"
-            (focus)="depth || reinitializeMenu()"
-          >
-            <ng-container *ngIf="!node.url">
-              {{ node.title }}
-            </ng-container>
-            <cx-icon [type]="iconType.CARET_DOWN"></cx-icon>
-          </button>
+          <ng-container *cxFeature="'!a11yButtonAriaFixes'">
+            <button
+              aria-level="4"
+              [attr.role]="(isDesktop$ | async) && depth ? 'heading' : 'button'"
+              [attr.aria-haspopup]="true"
+              [attr.aria-expanded]="false"
+              [attr.aria-controls]="node.title"
+              [attr.aria-describedby]="getAriaDescribedbyOfButton(node)"
```

Comment:
You've added it for negated cxFeature therefore if a11yButtonAriaFixes is enabled this won't be used. 
However when I change it on my local it solves the previous errors, but there is one more new one:
<img width="699" alt="image" src="https://github.com/user-attachments/assets/d47257c8-ed98-4c32-9afb-1caaeff4f602" />